internal/envoy/v3/auth.go

```diff
@@ -81,6 +81,7 @@ func validationContext(ca []byte, subjectName string, skipVerifyPeerCert bool) *
 	}
 
 	if len(subjectName) > 0 {
+		// nolint:staticcheck
```

Comment:
How will we make sure we come back and take these nolints out later? Should we have another TODO-style comment here or something?

Reply:
added https://github.com/projectcontour/contour/issues/4275 to the milestone and just added a note as a reminder to remove